contracts/ServiceRegistryV2.sol

```diff
@@ -413,10 +414,32 @@ contract ServiceRegistryV2 is Initializable, UUPSUpgradeable, OwnableUpgradeable
         return id;
     }
 
+    // =========================== Overrides ==============================
+
+    function _msgSender()
```

Comment:
Because the functions `_msgSender()` and `_msgData()` are inherited from both `ERC2771RecipientUpgradeable` and `AccessControlUpgradeable`, so it's mandatory to override otherwise it won't compile. In our case we just need the one from `ERC2771RecipientUpgradeable`.